docs/01-fundamentals/design-tokens.config.yml

```
@@ -48,4 +48,5 @@ context:
     $font-stack-base: "Inter, X-LocaleSpecific, sans-serif"
     $font-stack-firefox: "Metropolis, Inter, X-LocaleSpecific, sans-serif"
     $font-stack-mono: "Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace"
-    $font-stack-mozilla: "'Zilla Slab', Inter, X-LocaleSpecific, sans-serif"
+    $font-stack-mozilla-text: "'Mozilla Text', Inter, 'Helvetica Neue', Arial, X-LocaleSpecific, sans-serif"
+    $font-stack-mozilla-headline: "'Mozilla Headline', Inter, 'Helvetica Neue', Arial, X-LocaleSpecific, sans-serif"
```

I missed this one:
```suggestion
    $font-stack-mozilla-text: "'Mozilla Text', 'Helvetica Neue', Arial, X-LocaleSpecific, sans-serif"
    $font-stack-mozilla-headline: "'Mozilla Headline', 'Helvetica Neue', Arial, X-LocaleSpecific, sans-serif"
```
removed in [protocol-tokens#116](https://github.com/mozilla/protocol-tokens/pull/116/files)

(I'm preparing a tokens.config doc update and a changelog entry for tokens version bump so I'll include this fix with it…)